src/app/introduction/activity-3/activity-3.component.ts

```diff
@@ -19,7 +20,10 @@ export class PicturePuzzleComponent implements OnInit {
     // "#video" is the name of the template variable in the video element
     @ViewChild('video') video: any;
 
-    private PUZZLE_DIFFICULTY = 4;
+    private PUZZLE_DIFFICULTY: number;
```

Comment:
This is not a read only variable. Please use camelCase.

Reply:
Done.